apps/meteor/server/startup/migrations/xrun.ts

```diff
@@ -1,11 +1,15 @@
 import { upsertPermissions } from '../../../app/authorization/server/functions/upsertPermissions';
 import { migrateDatabase, onServerVersionChange } from '../../lib/migrations';
+import { ensureCloudWorkspaceRegistered } from '../cloudRegistration';
 
 const { MIGRATION_VERSION = 'latest' } = process.env;
 
 const [version, ...subcommands] = MIGRATION_VERSION.split(',');
 
 await migrateDatabase(version === 'latest' ? version : parseInt(version), subcommands);
 
-// if the server is starting with a different version we update the permissions
-await onServerVersionChange(() => upsertPermissions());
+// perform operations when the server is starting with a different version
```

Comment:
I think that you can remove this comment, since it's in the `startup` folder and due to the function name, it's self explained